Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -1219,46 +1302,95 @@ WarpXOpenPMDPlot::WriteOpenPMDFieldsAll ( //const std::string& filename,
     if ( (0 == lev) && first_write_to_iteration )
       SetupFields(meshes, full_geom);
 
-    amrex::Box const & global_box = full_geom.Domain();
-    auto const global_size = getReversedVec(global_box.size());
+        amrex::Box const & global_box = full_geom.Domain();
```

Comment:
alignment here is off, needs to be on same level as `if` above

Reply:
ok, that is fixed.  It was an artifact of an old tab = 2 spaces setting in my editor